DuckDuckGo/TabViewControllerLongPressMenuExtension.swift

```diff
@@ -87,9 +102,58 @@ extension TabViewController {
         UIPasteboard.general.string = copyText
     }
     
-    private func onShareAction(forUrl url: URL, atPoint point: Point) {
+    private func onShareAction(forUrl url: URL, atPoint point: Point?) {
         Pixel.fire(pixel: .longPressMenuShareItem)
         guard let webView = webView else { return }
         presentShareSheet(withItems: [url], fromView: webView, atPoint: point)
     }
 }
+
+@available(iOS 13.0, *)
+extension TabViewController {
+
+    static let excludedLongPressItems = [
+        UIImage(systemName: "safari"),
+        UIImage(systemName: "eyeglasses"),
+        nil // hide/show link previews
```

Comment:
I think this is non-nil on iPad running iOS 13.0 (at least on simulator):
<img width="152" alt="prev" src="https://user-images.githubusercontent.com/2865434/66838632-4af7cd00-ef65-11e9-962d-b270c4a76427.png">